ui-modules/blueprint-composer/app/components/not-ready-appliance/not-ready-appliance.directive.js

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
philosophical question : for brand new code created by us, is it still the apache license that apply (serious question, I have no idea)

Reply:
I have asked some other dev (CK members) and it seems that we should keep the apache license. We can discuss this if necessary ;)